static/src/js/app/api.js

```diff
@@ -6,30 +6,32 @@ axios.defaults.xsrfCookieName = 'csrftoken';
 const BASE_URL = '/api/v1/';
 
 export default {
+  /* -------------------------------------------------------------------------- */
+  /*                               hedera.Profile                               */
+  /* -------------------------------------------------------------------------- */
```

Comment:
Thanks for the pointer to the *Comment Divider* VSCode plugin. That will help in maintaining/extending these divisions.